tests/integration/jinad/test_simple_distributed/test_integration.sh

```diff
@@ -0,0 +1,35 @@
+set -e
+
+if [ "${PWD##*/}" != "jina" ]
+  then
+    echo "test_integration.sh should only be run from the jina base directory"
+    exit 1
+fi
+
+# Setting env variables locals for this script
+export $(grep -v '^#' tests/distributed/.env | xargs -d '\n')
+
+docker-compose -f tests/distributed/docker-compose.yml --project-directory . up  --build -d
+
+sleep 5
+
+FLOW_ID=$(curl -s --request PUT "http://localhost:8000/v1/flow/yaml" \
+    -H  "accept: application/json" \
+    -H  "Content-Type: multipart/form-data" \
+    -F "uses_files=@pods/encode.yml" \
+    -F "uses_files=@pods/extract.yml" \
+    -F "uses_files=@pods/index.yml" \
+    -F "pymodules_files=@pods/text_loader.py" \
+    -F "yamlspec=@tests/distributed/flow-query.yml"\
+    | jq -r .flow_id)
+
+echo "Successfully started the flow: ${FLOW_ID}"
+
+curl --request POST -d '{"top_k": 10, "data": ["text:hey, dude"]}' -H 'Content-Type: application/json' '0.0.0.0:45678/api/search' | \
```

Comment:
In this case the only thing we can expect is that we get a document in output with `text` equals to `hey, dude`

Reply:
done